.env

```diff
@@ -4,3 +4,4 @@ CONFLUENCE_VERSION=6.0.7
 POSTGRESQL_VERSION=9.6
 JAVA_VERSION=zulu@1.8.232
 DATABASE=postgres
+COMPOSE_PROJECT_NAME=default
```

Comment:
Maybe this is not required as if you don't set any name, a default internal one is always used. Although including this env var could help noticing you can use it. 🤷